src/containers/PipelineRuns/PipelineRuns.test.js

```diff
@@ -161,6 +161,7 @@ beforeEach(() => {
   jest.spyOn(API, 'getServiceAccounts').mockImplementation(() => []);
   jest.spyOn(API, 'getPipelines').mockImplementation(() => []);
   jest.spyOn(API, 'getPipelineResources').mockImplementation(() => []);
+  jest.spyOn(API, 'rerunPipelineRun').mockImplementation(() => []);
```

Comment:
can we spy on this in the test where it's actually used or is it needed across multiple tests?